packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -260,22 +218,85 @@ HeaderBar.Help = Help;
 HeaderBar.User = User;
 HeaderBar.Products = Products;
 
-HeaderBar.propTypes = {
-	logo: PropTypes.shape(Logo.propTypes),
-	brand: PropTypes.shape(Brand.propTypes),
-	env: PropTypes.shape(Environment.propTypes),
-	search: PropTypes.shape(Search.propTypes),
-	help: PropTypes.shape(Help.propTypes),
-	user: PropTypes.shape(User.propTypes),
-	products: PropTypes.shape(Products.propTypes),
-	renderers: PropTypes.shape({
-		Logo: PropTypes.func,
-		Brand: PropTypes.func,
-		Environment: PropTypes.func,
-		Search: PropTypes.func,
-		User: PropTypes.func,
-		Products: PropTypes.func,
-	}),
+HeaderBar.defaultProps = {
+	t: defaultTranslateFn,
+};
+
+if (process.env.NODE_ENV !== 'production') {
+	Logo.propTypes = {
+		isFull: PropTypes.bool,
+		renderers: PropTypes.shape({
+			Action: PropTypes.func,
+		}),
+		t: PropTypes.func.isRequired,
+	};
+
+	Brand.propTypes = {
+		isSeparated: PropTypes.bool,
+		renderers: PropTypes.shape({
+			Action: PropTypes.func,
+		}),
+	};
+
+	Environment.propTypes = {
+		renderers: PropTypes.shape({
+			ActionDropdown: PropTypes.func,
+		}),
+	};
+
+	Search.propTypes = {
+		...Typeahead.propTypes,
+		renderers: PropTypes.shape({
+			Typeahead: PropTypes.func,
+		}),
+	};
+
+	Help.propTypes = {
+		renderers: PropTypes.shape({
+			ActionSplitDropdown: PropTypes.func,
+			Action: PropTypes.func,
+		}),
+		t: PropTypes.func.isRequired,
+	};
+
+	User.propTypes = {
+		renderers: PropTypes.shape({
+			ActionDropdown: PropTypes.func,
+			name: PropTypes.string.isRequired,
+			firstName: PropTypes.string,
+			lastName: PropTypes.string,
+		}),
+	};
+
+	Products.propTypes = {
+		renderers: PropTypes.shape({
+			ActionDropdown: PropTypes.func,
+		}),
+		t: PropTypes.func.isRequired,
+	};
+
+	HeaderBar.propTypes = {
+		logo: PropTypes.shape(Logo.propTypes),
+		brand: PropTypes.shape(Brand.propTypes),
+		env: PropTypes.shape(Environment.propTypes),
+		search: PropTypes.shape(Search.propTypes),
+		help: PropTypes.shape(Help.propTypes),
+		user: PropTypes.shape(User.propTypes),
+		products: PropTypes.shape(Products.propTypes),
+		renderers: PropTypes.shape({
+			Logo: PropTypes.func,
+			Brand: PropTypes.func,
+			Environment: PropTypes.func,
+			Search: PropTypes.func,
+			User: PropTypes.func,
+			Products: PropTypes.func,
+		}),
+		t: PropTypes.func,
+	};
+}
+
+export {
+	HeaderBar,
```

Comment:
Still the same question : why export the non translated component as it should behave the same as translated one without conf